Sources/Web3ModalUI/Components/W3MChipButtonStyle.swift

```diff
@@ -111,6 +111,7 @@ public struct W3MChipButtonStyle<LeadingImageContent: View, TrailingImageContent
         .padding(.vertical, verticalPadding)
         .padding(.leading, leadingPadding)
         .padding(.trailing, trailingPadding)
+        .frame(minHeight: size == .m ? 36 : 32)
```

Comment:
hardcode?

Reply:
Unfortunately this is not using predefined values